tools/wasme/cli/docs/content/tutorial_code/wasme_operator.md

```diff
@@ -215,18 +217,24 @@ status:
       state: Succeeded
     productpage-v1:
       state: Succeeded
-    ratings-v1:
-      state: Succeeded
-    reviews-v1:
-      state: Succeeded
-    reviews-v2:
-      state: Succeeded
-    reviews-v3:
-      state: Succeeded
```

Comment:
The filter should still be deployed to the ratings and reviews services following this guide.

Reply:
Spoke offline about this one - The yaml snipped used in this tutorial _only_ installs productpage and details, as they are all that's required to demonstrate this working. I'm good with removing this bit from the example.